src/admin-views/editor/fieldset/advanced.php

```diff
@@ -52,6 +52,24 @@
 			class="ticket_field ticket_form_right"
 			id="ticket_description"
 		><?php echo esc_textarea( $ticket ? $ticket->description : null ) ?></textarea>
+		<div class="input_block">
+			<label class="tribe_soft_note">
+				<input
+						type="checkbox"
+						id="tribe_tickets_show_type"
+						name="ticket_show_type"
+						value="1"
+						class="ticket_field ticket_form_left"
+						<?php checked( true, $ticket ? $ticket->show_type : true ); ?>
+				>
+				<?php
+				echo esc_html( sprintf(
+						__( 'Show type on front end %s form.', 'event-tickets' ),
```

Comment:
* :no_entry_sign: Multi-line function call not indented correctly; expected 20 spaces but found 24
* :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 8ac7c8ff1a4ef8b0e97ffd09f7410c38</sub>

Reply:
Let's address these two.